### PR TITLE
update_version.sh: automatic version update in source

### PR DESCRIPTION
**Pull request by caisq**
Usage example:
  update_version.sh 0.8.0

### CODE REVIEW

File: tensorflow/tools/ci_build/update_version.sh [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+# ==============================================================================
+# 
+# Automatically update TensorFlow version in source files
+# 
+# Usage:  update_version.sh <new_major_ver>.<new_minor_ver>.<new_patch_ver>
+#         e.g.,
+#           update_version 0.7.2
+
+# Helper functions
+check_existence() {
```
Comment (martinwicke) [inline tensorflow/tools/ci_build/update_version.sh]: What does this add over test?

test -d $1 || die "Error..."

**Comment by martinwicke** [comment]
Yes!

**Comment by caisq** [comment]
Addressed @martinwicke's comment. Added safety check for lingering old version strings and warning if those are found.

**Comment by martinwicke** [comment]
merged.

**Pull request closed without merging**